packages/block-library/src/site-logo/editor.scss

```diff
@@ -151,3 +151,25 @@
 		height: $grid-unit-50;
 	}
 }
+
+// This is necessary for the editor resize handles to accurately work on a non-floated, non-resized logo.
+.wp-block-site-logo .components-resizable-box__container {
+	// Using "display: table" because:
+	// - it visually hides empty white space in between elements
+	// - it allows the element to be as wide as its contents (instead of 100% width, as it would be with `display: block`)
+	display: table;
+	img {
+		display: block;
+		width: inherit;
+	}
+}
+
+.wp-block-site-logo.has-custom-border {
+	.reactEasyCrop_CropArea {
+		border-width: 0;
+	}
```

Comment:
> > The image becomes slightly larger when the crop tool is enabled (This also seems to be happening with image blocks).
> 
> I am able to solve this by removing an outer border from the crop area ( .reactEasyCrop_CropArea). But when I do that, the grey background of the cropping tool is visible on the right and left side of the image.

Is it possible to delete this style?

The problem of the image becoming slightly larger when the crop tool is enabled seems to be resolved simply by the following style, which has already been added in this PR.

```scss
.wp-block-image__crop-area {
	box-sizing: border-box;
}
```

Reply:
I didn't like the result of removing the border width, but without it I still do see the size difference.

Reply:
I also don't think the blocks should override these type of styles; the crop tool itself may need updates.